### DISCUSSION TITLE
Use `arrow` library

### DISCUSSION THREAD

**Discussion by brianmhunt** [collective/icalendar#544] [Ideas]
I just looked through the code, and if you're refactoring for 4.0 already you might be interested in the [Arrow library](http://crsmithdev.com/arrow/).

Arrow is probably nothing new, and I haven't looked too deeply at the code, but it looks like it could save a bunch of complexity and perhaps reduce some library dependencies (e.g. dateutil, datetime, timedelta, pytz, etc).

I just brought it up since it's not mentioned in a search of the issues here.  No need to keep this issue open if Arrow isn't a plausible direction for the library. :)

:beers:

**Comment by equaeghe** [comment]
Current url: https://arrow.readthedocs.io/
